storybook/stories/API/cartesian/YAxis.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { ComponentProps, useState } from 'react';
+import { YAxis, XAxis, Legend, ResponsiveContainer, ComposedChart, Bar } from '../../../../src';
+import { pageData } from '../../data';
+
+export default {
+  component: YAxis,
+  argTypes: {
+    stroke: {
+      control: { type: 'color' },
+    },
+  },
+};
+
+const data = [
+  { name: 'Page A', pv: 800, amt: 1400 },
+  { name: 'Page B', pv: 967, amt: 1506 },
+  { name: 'Page C', pv: 1098, amt: 989 },
+  { name: 'Page D', pv: 1200, amt: 1228 },
+  { name: 'Page E', pv: 1108, amt: 1100 },
+  { name: 'Page F', pv: 680, amt: 1700 },
+];
+
+export const Simple = {
```

Comment:
Can you add some text to both stories indicating that the legend can be clicked to toggle data visibility?

For the IncludeHidden story can you make the text tell the viewer what the difference between simple and includeHidden is?

Reply:
Just had a quick look at doing this and I can't see any other stories with text. Do you have a preferred place or way to add some?

Reply:
Have a look at the `storybook/stories/API/cartesian/Line.stories.tsx` for example.

Reply:
Thanks for the pointer! I've just added some notes to this effect.